### DISCUSSION TITLE
SEO issue on documentation

### DISCUSSION THREAD

**Discussion by CarolinePublipresse** [OFFLINE-GmbH/oc-boxes-support#51] [Ideas and feature requests]
I often use google to search on documentation and check if I'm searching in the right section the information I need. 

I was surprised to see that google has not registered any internal page, despite the fact that you have provided a full documentation, well organized on diffrent sections, and pages, with different urls and anchors : 
[site:docs.boxes.offline.ch](https://www.google.com/search?q=site%3Adocs.boxes.offline.ch&rlz=1C1CHBF_frFR932FR932&oq=site%3Adocs.boxes.offline.ch&aqs=chrome..69i57j69i58.7967j0j4&sourceid=chrome&ie=UTF-8)

So I can't use it to search for all occurences of "validation" for exemple, if I want to see all examples of codes using validation... 
It would be nicer for us to be able to use it.

**Comment by tobias-kuendig** [comment]
Thank you for the heads-up @CarolinePublipresse! 

I have submitted a proper Sitemap for the docs website to Google. The indexing process should start shortly.
